SIT.Manager/Services/AppUpdaterService.cs

```diff
@@ -31,11 +31,11 @@ private static string ProcessName
         {
             if (OperatingSystem.IsWindows())
             {
-                return "SIT.Manager.Avalonia.Desktop.exe";
+                return "SIT.Manager.Desktop.exe";
             }
             else if (OperatingSystem.IsLinux())
             {
-                return "SIT.Manager.Avalonia.Desktop";
+                return "SIT.Manager.Desktop";
```

Comment:
Are these executable names going to be right?

Reply:
Thanks for that catch. I ran the find and replace before I decided to rename the executable